hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/ImmutableMemStoreLAB.java

```diff
@@ -138,5 +138,16 @@ public boolean isOffHeap() {
     return ChunkCreator.getInstance().isOffheap();
   }
 
+  /**
+   * This method is only used for test.
+   */
+  public int getRefCntValue() {
```

Comment:
Please use RestrictedApi in error prone annotation, it will cause an compile error in our pre commit build if you have the annotation but someone breaks the rule.

Reply:
Ok, already used RestrictedApi to fix the problem.